src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -81,20 +87,28 @@ def __init__(
         do_normalize=True,
         image_mean=None,
         image_std=None,
-        ignore_index=255,
+        ignore_index=None,
+        reduce_labels=False,
```

Comment:
Would update the order here, first `reduce_labels`, cause in case that's True, it will use the `ignore_index`

Reply:
`ignore_index` may be used even if `recude_labels` is False